Calibration/LumiAlCaRecoProducers/plugins/RawPCCProducer.cc

```diff
@@ -40,42 +40,44 @@ class RawPCCProducer : public edm::global::EDProducer<edm::EndLuminosityBlockPro
   void globalEndLuminosityBlockProduce(edm::LuminosityBlock& lumiSeg, const edm::EventSetup& iSetup) const final;
   void produce(edm::StreamID, edm::Event& iEvent, const edm::EventSetup& iSetup) const final;
 
-  edm::EDGetTokenT<reco::PixelClusterCounts> pccToken_;
-  const edm::ESGetToken<LumiCorrections, LumiCorrectionsRcd> lumiCorrectionsToken_;
-  const edm::EDPutTokenT<LumiInfo> putToken_;
+  edm::EDGetTokenT<reco::PixelClusterCounts> pccToken_;                              //input object labels
```

Comment:
```suggestion
  const edm::EDGetTokenT<reco::PixelClusterCounts> pccToken_;                              //input object labels
```

Reply:
hi Tamas. This change did not work. See the code-checks compilation error below. I guess we would need to change something about the line: 
pccToken_ =  consumes<reco::PixelClusterCounts,  ... 
For now I will remove the "const " again.